cpp/FileTransferAgent.cpp

```diff
@@ -300,7 +300,8 @@ void Snowflake::Client::FileTransferAgent::compressSourceFile(
 {
   CXX_LOG_DEBUG("Starting file compression");
   
-  char tempDir[100];
+  char tempDir[150]={0};
```

Comment:
can this be larger? 150 is too small for full path. could this be 4096?

Reply:
Was adding extra 37 bytes of UUID so thought +50 should do. But yeah customers can have long path. Will fix it.